packages/flutter/test/widgets/reorderable_list_test.dart

```diff
@@ -587,37 +630,34 @@ void main() {
   });
 }
 
-class TestList extends StatefulWidget {
+class TestList extends StatelessWidget {
```

Comment:
Making TestList stateless because it was never needed in the first place?

Reply:
As near as I could tell, it wasn't. There was no internal state kept and all the tests passed after making it stateless so I thought it would be better to just keep it simpler.